server/@types/viewModels/index.d.ts

```diff
@@ -20,17 +20,31 @@ declare module 'viewModels' {
     hasSupportNeeds: boolean
   }
 
-  export interface Pagination {
-    totalElements: number
-    totalPages: number
-    page: number
-    last: boolean
-    first: boolean
-    pageSize: number
-  }
-
   export interface PrisonerSearch {
-    pagination: Pagination
+    pagination: MojPaginationParams
     prisoners: PrisonerSearchSummary[]
   }
+
+  export interface MojPaginationParams {
```

Comment:
This type is the exact type/structure that the MoJ Pagination Component expects to be passed to it (not sure why they didnt publish it as a type as well as the component itself, but never mind 🤷‍♂️ )

https://design-patterns.service.justice.gov.uk/components/pagination/